### PR TITLE
Add github-accessors_test.go

### PR DESCRIPTION
**Pull request by gmlewis**
While taking a look at code coverage, I noticed that the auto-generated `github-accessors.go` file had 0.3% code coverage.

This PR increases the `github-accessors.go` code coverage from 0.3% to 100%.

### CODE REVIEW

**Comment by gmlewis** [comment]
Apparently, codecov.io ignores `github-accessors.go`, but this PR still benefits the local code coverage results, so I think it is still a good idea to include.

File: github/github-accessors_test.go [hunk 1]
```diff
@@ -0,0 +1,18304 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
```
Comment (nightlark) [inline github/github-accessors_test.go:1]: Should the copyright year get updated based on when these files are (re)generated?

Comment (gmlewis) [reply github/github-accessors_test.go:1]: Good question, but the answer is no. Once a file is created, the copyright year remains unmodified.
(At least for this repo, that is the instruction given from the Google Open Source team.)

Comment (gmlewis) [reply github/github-accessors_test.go:1]: Oh!  I see your point now.  This is a new test file.

OK, I'm going to make an executive decision and keep the two sync'd to the same 2017 year.

Thank you, @nightlark .

**Pull request merged**